modules/sizing-requirements-for-gitops.adoc

```diff
@@ -28,3 +28,47 @@ Optionally, you can also use the ArgoCD custom resource with the `oc` command to
 ----
 oc edit argocd <name of argo cd> -n namespace
 ----
+
+[id="sizing-requirements-for-argocd-redis_{context}"]
+== Sizing requirements for argocd-redis
+
+During the capacity planning stage for your application in the {gitops-title} Operator, you must ensure that an adequate amount of resources such as memory, CPU, and storage are allocated for the `argocd-redis` pod.
+
+The default memory limit for the `argocd-redis` caching server is `256M`. Memory limit is a value that constitutes a fixed amount of memory that can be allocated across all the processes in a pod.
```

Comment:
```suggestion

```

Reply:
I am not sure why we are giving an explanation of what memory limit is here. I does not fit here in my opinion. If we want to provide it somewhere, then maybe in the previous section?

Also, we do not need to state the default limit for Redis here as it is stated in the table above this section, so I believe having this information here again is not necessarry.

Reply:
Ok, that makes sense. I will remove this sentence. Fixed.